test/json_formatter_SUITE.erl

```diff
@@ -346,21 +346,18 @@ large_event_dont_crash_formatter(_Config) ->
 %%
 
 example_acc(Body) ->
+    Elem = {xmlel, <<"message">>,
+            [{<<"type">>, <<"chat">>}, {<<"id">>, <<"1111">>}],
+            [{xmlel, <<"body">>,[], [{xmlcdata, Body}]}]},
```

Comment:
Very nit-picky, I can merge as it is.
```suggestion
            [{xmlel, <<"body">>, [], [{xmlcdata, Body}]}]},
```

Reply:
Hmmmm I think we can spare an extra CI run here, there were no spaces before, I tried adding them and missed one... 😅